Project_source/app/src/main/java/android/clicker/school_live_simulator/Game.kt

```diff
@@ -1,4 +1,41 @@
 package android.clicker.school_live_simulator
 
-class Game {
+object Game {
+    private var difficulty_state: GameDifficultyState = NormalMode()
+    private var save_handlers: ArrayList<function> = ArrayListOf()
+    private var load_handlers: ArrayList<function> = ArrayListOf()
+
+    private lateinit var player: InGameWrapper
```

Comment:
It is a deprecated declaration. Now it should have Player type.

Reply:
Fixed